lib/render/threejs/three-texture-loader-browser.ts

```diff
@@ -84,14 +84,18 @@ async function load(mimeType: string, objectUrl: string): Promise<ThreeTexture>
 	if (mimeType === 'application/octet-stream') {
 		return await loadHdrTexture(objectUrl);
 	}
-	return Promise.resolve(loadLdrTexture(objectUrl));
+	return loadLdrTexture(objectUrl);
 }
 
-function loadLdrTexture(objectUrl: string): ThreeTexture {
-	const texture = new ThreeTexture();
-	texture.image = new Image();
-	texture.image.src = objectUrl;
-	return texture;
```

Comment:
about your comment
>three will know when it's loaded and handle it at that point
in theory yes, there are two points against it:

- its a warning and should be avoided if possible (our app does a lot when bootstrapping - so lets use a promise and don't let threejs wait in a busy loop polling until the image arrives)
- i see more than 9000 lines of exact the same log message when this error happens (the exported console log is more than 13MB) - which makes not really fun debugging ;)

Reply:
Fair enough. I admit I haven't tested the other way, if you're saying this doesn't come with a performance hit, I'm fine with it.

Reply:
jep - never saw this on my beefy machine but on my aging home notebook